editor/filesystem_dock.cpp

```diff
@@ -2690,8 +2690,8 @@ FileSystemDock::FileSystemDock(EditorNode *p_editor) {
 	// `KEY_MASK_CMD | KEY_C` conflicts with other editor shortcuts.
 	ED_SHORTCUT("filesystem_dock/copy_path", TTR("Copy Path"), KEY_MASK_CMD | KEY_MASK_SHIFT | KEY_C);
 	ED_SHORTCUT("filesystem_dock/duplicate", TTR("Duplicate..."), KEY_MASK_CMD | KEY_D);
-	ED_SHORTCUT("filesystem_dock/delete", TTR("Delete"), KEY_DELETE);
-	ED_SHORTCUT("filesystem_dock/rename", TTR("Rename"), KEY_F2);
+	ED_SHORTCUT("filesystem_dock/delete", TTR("Move to trash"), KEY_DELETE);
```

Comment:
Much to my dislike, we use Title Case for menu entries:
```suggestion
	ED_SHORTCUT("filesystem_dock/delete", TTR("Move to Trash"), KEY_DELETE);
```

Reply:
Oh damn I saw that and thought I fixed that before I pushed, sorry.